build/src/main/resources/agent.properties

```diff
@@ -225,6 +225,20 @@ reporter.outputServer.tls.key=
 reporter.outputServer.tls.cert=
 reporter.outputServer.tls.ca_cert=
 
+
+# --- redefine to output properties
+reporter.log.output.messageMaxBytes=999900
+reporter.log.output.reportThread=1
+reporter.log.output.queuedMaxSpans=1000
+reporter.log.output.queuedMaxSize=1000000
+reporter.log.output.messageTimeout=1000
+
```

Comment:
add some doc:
## sender.appendType config
## [http] send to http server
## [kafka] send to kafka
## [console] send to console